### ISSUE TITLE
Installation Assistant fails with 504 Gateway Time-out

### ISSUE THREAD

**Issue by h3ndrik** [YunoHost-Apps/prestashop_ynh#16]
no matter what i do, the configuration assistant always fails on the last step 'Store installation' at about 12% with the following error:

```
An error occurred during installation...
You can use the links on the left column to go back to the previous steps, or restart the installation process by clicking here.
1: HTTP 504 - error -
504 Gateway Time-out
nginx
```

I installed prestashop to a new domain, choosing `/` as the path.

Already tried with or without demo data, different languages and clearing the cookies/data in my browser. It connects to mysql and proceeds to create several tables, but that's it. I had a look at the psphpinfo.php (mentioned somewhere in the prestashop docs) and all php settings seem to be fine.

**Comment by ericgaspar** [comment]
I have updated the package to the latest version; I am able to install it and managed to finish the post install successfully. However, there are still some issues to be resolved (i.e. access to administration page blocked by NGINX)

**Comment by valterbarreira** [comment]
It also gives me the same error

**Comment by valterbarreira** [comment]
I have sucessfull installed the package, but now i have an error regarding upload of a theme file. I'll open a new issue
